Fix token exchange required features (#1305)
keycloak/keycloak-documentation: securing_apps/topics/token-exchange/token-exchange.adoc

include::../templates/techpreview.adoc[]

[NOTE]

> In order to use more than the link:#internal-token-to-internal-token-exchange[Internal Token to Internal Token Exchange] flow, you should also enable the `admin_fine_grained_authz` feature. Please, take a look at link:{installguide_profile_link}[{installguide_profile_name}]. 

Reviewer: ```suggestion
To use more than the link:#internal-token-to-internal-token-exchange[Internal Token to Internal Token Exchange] flow, you should also enable the `admin_fine_grained_authz` feature. For details, see  link:{installguide_profile_link}[{installguide_profile_name}]. 
```
